english/total-war/war-start.md

```diff
@@ -0,0 +1,7 @@
+Total War - Warhammer is a turn-based strategy game with epic real-time tactical battles between armies.
+
+It is also the first in its franchise that introduces a fantasy setting from the [Warhammer universe](war-start/universe/universe.md).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.